src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -1719,5 +1719,47 @@ INamedTypeSymbolInternal INamedTypeSymbolInternal.EnumUnderlyingType
                 return this.EnumUnderlyingType;
             }
         }
+
+        /// <summary>
+        /// Produce all inherited base extensions in topologically sorted order. We use
+        /// TypeSymbol.BaseExtensionsNoUseSiteDiagnostics as the source of edge data,
+        /// Produce all base extensions in topologically sorted order. We use
```

Comment:
```suggestion
```

Reply:
I didn't follow. The suggested change looks like it'll remove the entire line.

Reply:
Sorry, seems my comment got lost. It looks to me like the line is almost a duplicate of the first line, inserted in the middle of another sentence.

Reply:
Got it. Fixed. Thanks!